code/modules/food_and_drink/plants.dm

```diff
@@ -512,7 +512,7 @@
 			hitMob.do_disorient(stamina_damage = 35, weakened = 0, stunned = 0, disorient = 30, remove_stamina_below_zero = 0)
 			hitMob.TakeDamageAccountArmor("chest", rand(damMin, damMax), 0)
 
-	throw_at(atom/target, range, speed)
+	throw_at(atom/target, range, speed, list/params, turf/thrown_from, throw_type = 1, allow_anchored = 0)
```

Comment:
you are accepting arguments here that won't ever get passed to the parent

parent call should either be `..()` to pass all or expanded to include the new arguments that you wish to send down and the others removed as they are unused in this child.